renderer/index.js

```diff
@@ -153,6 +153,11 @@ function cleanUpConfig () {
       delete ts.posterURL
       ts.posterFileName = infoHash + extension
     }
+
+    // Migration: add per-file selections
+    if (!ts.selections) {
+      ts.selections = ts.files.map((x) => true)
```

Comment:
Yeah for sure wasn't going to do it now.

I actually think it's fine --- if we put everything into `info` then things will become even longer. For example

```
state.saved.torrents[0].files[0].name
```
...would turn into

```
state.saved.torrents[0].info.files[0].name
```